test/uk/gov/hmrc/uknwauthcheckerapi/controllers/AuthorisationsControllerSpec.scala

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmrc.uknwauthcheckerapi.controllers
+
+import org.apache.pekko.actor.setup.Setup
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.libs.json.Json
+import play.api.mvc.Result
+import play.api.test.Helpers._
+import play.api.test.{FakeRequest, Helpers}
+import uk.gov.hmrc.uknwauthcheckerapi.models.{AuthorisationRequest, AuthorisationsResponse, EISResult}
+
+import scala.concurrent.Future
+
+class AuthorisationsControllerSpec extends AnyWordSpec with Matchers {
+
+  private val controller = new AuthorisationsController(Helpers.stubControllerComponents())
+
+  "AuthorisationsController" should {
+
+    val converted =
+      AuthorisationsResponse(
+        "01-01-1999",
+        Array(
+          EISResult("SampleEORI1", authorised = true),
+          EISResult("SampleEORI2", authorised = true)
+        )
+      )
+
+    "return OK" in new Setup {
+      val request: FakeRequest[AuthorisationRequest] =
+        FakeRequest().withBody(AuthorisationRequest("01-01-1999", Array("SampleEORI1", "SampleEORI2")))
```

Comment:
Use an arbitrary for the date.

Use a generator which can use a regular expression for EORI's

https://github.com/wolfendale/scalacheck-gen-regexp

https://github.com/hmrc/check-eori-number-frontend/blob/69b21a87a4f592238eef74ad22caeb5905eef685/app/uk/gov/hmrc/checkeorinumberfrontend/controllers/CheckEoriNumberController.scala#L69

Reply:
Introduced generator for localDate and eori